packages/fxa-content-server/app/scripts/lib/survey-targeter.js

```diff
@@ -69,16 +69,20 @@ export default class SurveyTargeter {
       }
 
       const selectedSurvey = this._selectSurvey(qualifiedSurveys);
+      const queryParamData = Object.assign(selectedSurvey.conditions, {
+        service: 'content',
```

Comment:
I might be a little confused here, I took the bug as passing along the `service` from content-server url, ex. `http://localhost:3030/reset_password?service=sync` to SurveyGizmo.

Reply:
Or by service you mean server?

Reply:
I did mean server, but used service to make it sound better since it's technically a publicly visible address -- happy to change to server to make everything look the same :)